G1ANT.Addon.OCR.Tesseract/G1ANT.Addon.OCR.Tesseract/Commands/OcrOfflineFindCommand.cs

```diff
@@ -37,31 +38,48 @@ public class Arguments : CommandArguments
 
             [Argument(Tooltip = "The language which should be considered trying to recognize text")]
             public TextStructure Language { get; set; } = new TextStructure("eng");
+
+            [Argument(Tooltip = "The ratio used for rescaling of the image before doing actual OCR. Default is 2.0. Higher values are better for recognizing details like small fonts.")]
+            public FloatStructure Sensitivity { get; set; } = new FloatStructure(2.0);
         }
         public OcrOfflineFindCommand(AbstractScripter scripter) : base(scripter)
         {
         }
 
-        double imgRescaleRatio = 4.0;
         public void Execute(Arguments arguments)
         {
-            var rectangle = !arguments.Relative.Value ? arguments.Area.Value : arguments.Area.Value.ToAbsoluteCoordinates();
+            if (!arguments.Area.Value.IsValidRectangle())
+                throw new ArgumentException("Argument Area is not a valid rectangle");
+            var rectangle = arguments.Area.Value;
+            if (arguments.Relative.Value)
+            {
+                var foregroundWindowRect = new RobotWin32.Rect();
+                RobotWin32.GetWindowRectangle(RobotWin32.GetForegroundWindow(), ref foregroundWindowRect);
+                rectangle = new Rectangle(rectangle.X + foregroundWindowRect.Left,
+                    rectangle.Y + foregroundWindowRect.Top,
+                    foregroundWindowRect.Right - foregroundWindowRect.Left,
+                    foregroundWindowRect.Bottom - foregroundWindowRect.Top);
+            }
             var partOfScreen = RobotWin32.GetPartOfScreen(rectangle);
             var language = arguments.Language.Value;
-            var search = arguments.Search.Value.ToLower();
-            var imgToParse = OcrOfflineHelper.RescaleImage(partOfScreen, imgRescaleRatio);
-            var imagePath = OcrOfflineHelper.SaveImageToTemporaryFolder(imgToParse);
+            var imgToParse = OcrOfflineHelper.RescaleImage(partOfScreen, arguments.Sensitivity.Value);
+            var search = arguments.Search.Value.ToLower().Trim();
             var dataPath = OcrOfflineHelper.GetResourcesFolder(language);
 
             try
             {
                 using (var tEngine = new TesseractEngine(dataPath, language, EngineMode.TesseractAndCube))
-                using (var img = Pix.LoadFromFile(imagePath))
+                using (var img = PixConverter.ToPix(imgToParse))
                 using (var page = tEngine.Process(img))
                 {
                     var rectResult = new Rectangle(-1, -1, -1, -1);
-                    var wordsWithRectPositions = GetWords(page.GetHOCRText(0));
-                    if (wordsWithRectPositions.ContainsValue(search))
+                    var wordsWithRectPositions = GetWords(page.GetHOCRText(0), arguments.Sensitivity.Value);
+                    var searchWords = search.Split(' ');
```

Comment:
Tooltip says "Provide only single words" is it still true? If not change tooltip (ask @Vegebutcher to confirm text if necessary)